input/docs/fundamentals/preprocessor-directives.md

```diff
@@ -140,3 +140,13 @@ The break directive, when placed on any line within your Cake file, will cause `
 ```bash
 #break
 ```
+
+# Using static directive
+The using static directive allows referencing a types static members without needed to specify the type name.
```

Comment:
Btw, shouldn't it be `type's`? Not sure since english confuses me sometimes 😄

Reply:
Yes it should be, let me fix it.

Reply:
Done.